packages/vue/src/components/FInteractiveTable/FInteractiveTable.vue

```diff
@@ -595,6 +593,16 @@ function setActiveRow(row: T | undefined): void {
                             * `row: ListItem;` The object to be visualized.
                         -->
                         <slot v-bind="{ row }" />
+
+                        <!--
+                            @slot Slot for screen reader text when row get focus. The row object is available through `v-slot="{ <propertyName> }"`, e.g. `v-slot="{ row }"`.
```

Comment:
Har skrivit det tidigare kopplat till tabell men jag ogillar att vi skriver ut hela "is available through v-slot [..]", dels för att det inte är det rekommenderade sättet (`#row-description="{ <propertyName }"` är) och du får html-validate fel av det men också för att jag inte tycker att vi ska ordbajsa grundläggande Vue-koncept.